local-cli/util/isPackagerRunning.js

```diff
@@ -19,8 +19,8 @@ const fetch = require('node-fetch');
  *   - `unrecognized`: one other process is running on the port we expect the
  *                     packager to be running.
  */
-function isPackagerRunning() {
-  return fetch('http://localhost:8081/status').then(
+function isPackagerRunning(packagerPort = (process.env.RCT_PACKAGER_PORT || 8081)) {
```

Comment:
Please note: I left the default here because it is possible for someone to use this function outside of the build scripts.